biz.aQute.repository/src/aQute/bnd/repository/maven/provider/MavenBndRepository.java

```diff
@@ -213,10 +221,13 @@ public PutResult put(InputStream stream, PutOptions options) throws Exception {
 						}
 					}
 				}
-				if (!binaryArchive.isSnapshot())
+				if (configuration.noupdateOnRelease() == false && !binaryArchive.isSnapshot())
```

Comment:
Can't this just be `!configuration.noupdateOnRelease()` instead of the `== false`?